package.json

```diff
@@ -37,23 +38,26 @@
     "walk-sync": "^2.0.2"
   },
   "devDependencies": {
+    "@jest/globals": "^28.1.2",
     "broccoli-test-helper": "^2.0.0",
-    "eslint": "^6.8.0",
-    "eslint-config-prettier": "^6.10.1",
-    "eslint-plugin-node": "^11.0.0",
-    "eslint-plugin-prettier": "^3.1.2",
+    "eslint": "^8.19.0",
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-prettier": "^4.2.1",
     "husky": "^4.2.3",
-    "jest": "^25.2.3",
+    "jest": "^28.1.2",
+    "jest-environment-node": "^28.1.2",
     "lint-staged": "^10.1.1",
+    "lodash": "^4.17.21",
     "npm-run-all": "^4.1.5",
     "prettier": "^2.0.2",
-    "release-it": "^14.0.0",
-    "release-it-lerna-changelog": "^3.0.0",
+    "release-it": "^15.1.1",
+    "release-it-lerna-changelog": "^4.0.1",
     "sinon": "^9.0.1",
     "tmp": "^0.1.0"
   },
   "peerDependencies": {
-    "release-it": "^14.0.0"
+    "release-it": "^15.0.0"
   },
   "engines": {
     "node": "10.* || 12.* || >= 14"
```

Comment:
I guess the set of supported node versions may need updating

Reply:
I updated them. Since this is a breaking change moving from CJS to ESM, it's reasonable to remove all non LTS versions of node.